src/vm/eventpipe.cpp

```diff
@@ -45,6 +45,12 @@ extern "C"
 #endif
 void InitProvidersAndEvents();
 
+// This is a wrapper method for LTTng. See https://github.com/dotnet/coreclr/pull/27273 for details.
+extern "C" bool XplatEventLoggerIsEnabled()
+{
+    return XplatEventLogger::IsEventLoggingEnabled();
+}
+
```

Comment:
This is really LTTng specific, and so it doesn't belong in eventpipe.cpp.  How about putting this in eventtrace.cpp?